packages/office-ui-fabric-react/src/components/Rating/Rating.base.tsx

```diff
@@ -53,18 +53,23 @@ export class RatingBase extends BaseComponent<IRatingProps, IRatingState> {
     max: 5
   };
   private _id: string;
+  private _min: number;
```

Comment:
you shouldn't be storing state in member variables. Changing the value (e.g. passing a new `min` prop will not do anything... this would be an antipattern. Reserve member variables for truely encapsulated, non input values that the class owns, that should not generate re-renders on changes.

Reply:
Right, this is the 'new min' (we can remove the min prop later, but this way if someone is using it their app won't break).  The new minimum is 1 or zero, still defaulting to 1, and set to zero by 'allowZeroStars', with a check to see if the min prop is set to anything but 1 to support the old way if used.

In any case, you don't want to reset the min after the component mounts, unless I'm missing some usage.